db/schema.rb

```diff
@@ -1114,6 +1115,12 @@
   add_index "scorecards", ["cross"], name: "index_scorecards_on_cross", using: :btree
   add_index "scorecards", ["ope"], name: "index_scorecards_on_ope", using: :btree
 
+  create_table "sec109_closed_schools", force: :cascade do |t|
```

Comment:
Where did this come from? It's not reflected in the migration

Reply:
branch not updated with master